distribution/changelog.txt

```diff
@@ -49,6 +50,7 @@
 - Fix: [#18606] JSON objects do not take priority over the DAT files they supersede.
 - Fix: [#18620] [Plugin] Crash when reading widget properties from windows that have both static and tab widgets.
 - Fix: [#18653] Negative ratings multipliers do not appear in Vehicle tab.
+- Fix: [#18675] [Plugin] Plugins get incorrect g2 image icons.
```

Comment:
```suggestion
```

The issue was introduced after the last release, so shouldn’t be in the changelog.